x-pack/package.json

```diff
@@ -231,7 +231,6 @@
     "file-type": "^10.9.0",
     "font-awesome": "4.7.0",
     "formsy-react": "^1.1.5",
-    "fp-ts": "^2.0.5",
```

Comment:
__note:__ I've just been told that the best practice in Kibana is to explicitly list dependencies in both root `package.json` and `xpack/package.json` if it's used in both OSS and in XPack plugins even though it's not strictly necessary. Would you mind still keeping these two here as well?

Reply:
Sure :+1: